mirage/models/visitor.js

```diff
@@ -0,0 +1,4 @@
+import { Model } from 'ember-cli-mirage';
+
+export default Model.extend({
+});
```

Comment:
You don't need to create this anymore. This is automatically figured out from the ember data model